File: 수업/C++/200702_class/Human/Human.cpp
#include "Human.h"
Human::Human(string n = "사람", int a = 0) {
	name = n; age = a;
}
Human::~Human() {
	cout << "Human 소멸자" << endl;
}
void Human::eat() {
	cout << name << "은 밥을 먹는다." << endl;
}
void Human::sleep() {
	cout << name << "은 잠을 잔다." << endl;
}
void Human::work() {
	cout << name << "은 일을 한다." << endl;
}
void Human::info() {
	cout << "이름 : " << name << endl;
	cout << "나이 : " << age << endl;
}

File: 수업/C#/200729_class/_40_Serialize/Human.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _40_Serialize {
    [Serializable]
    class Human {
        public  string Name { get; set; }
        public int Age { get; set; }
        public float Height { get; set; }
        public float Weight { get; set; }
        public override string ToString() {
            return Name + " : " + Age + " : " + Height + " : " + Weight;
        }
    }
}
